notifications/build-tools/esplugin-coverage.gradle

```diff
@@ -58,7 +58,7 @@ integTest {
 
 jacocoTestReport {
     dependsOn integTest, test
-    executionData dummyTest.jacoco.destinationFile, dummyIntegTest.jacoco.destinationFile
```

Comment:
why integ test is excluded?

Reply:
the existing trick of dumping IT test result to jacoco doesn't seem working fully. Also, elasticsearch official doc says coverage is not doable https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc#test-coverage-analysis
So currently only UT is dumped into jacoco report. I added a comment in the code for us to look back